### ISSUE TITLE
Xbox One gamepad "View" button is not detected on android 12+

### ISSUE THREAD

**Issue by dannygaray60** [godotengine/godot#67920]
### Godot version

3.5.1 stable

### System information

Android 12 - Samsung A12

### Issue description

On android 12+, the "view" button doesn't work.

### Steps to reproduce

Create a new Godot Project and assign an action to view button (select-playstation, "-" button on switch). Export and install on android (I tested on Android 12) It's the only button that doesn't work.

**Comment by Calinou** [comment]
Does the same button work in other Android games? Does it work on desktop platforms?

**Comment by dannygaray60** [comment]
@Calinou yes, the button is working in other games, and is working in old godot version (I tested in a game made with godot 3.3.3). On PC platforms there are no problems

**Comment by Torguen** [comment]
@dannygaray60 
Hey, could you please try the bug I reported for android?
This bug needs to be checked.
https://github.com/godotengine/godot/issues/67127

**Comment by dannygaray60** [comment]
@Torguen on Android 12 and with a Xbox One Gamepad (2nd gen, without Share button) the axis works fine. #67127

**Comment by dannygaray60** [comment]
Tested in Godot 4.0 stable + Android 13. The view button doesn't works

**Comment by dannygaray60** [comment]
Any way to patch this?

**Comment by dannygaray60** [comment]
Tested with Android13 and Xbox Series X controller, have the same issue in Godot4.2 beta1...

**Comment by dannygaray60** [comment]
Tried with Godot 3.6beta5 and the result is the same... however the "share" button is detected, but show this error:

```
E 0:08:20.811   joy_button: Index p_button = -38 is out of bounds (JOY_BUTTON_MAX = 128).
  <C++ Source>  main/input_default.cpp:888 @ joy_button()
```

But with the "view" button nothing happens. Is this a problem of godot or SDLmapping list?

**Comment by Alex2782** [comment]
@dannygaray60 please try PR #106021 with V4.4+
_I could also create a custom_template for you if you are not familiar with compiling._

**Comment by Alex2782** [comment]
@dannygaray60 [android_debug_xbox.apk](https://drive.google.com/file/d/1_IYrmbOmWVu16KLvNhQ_4sau2bSq-scK/view?usp=drive_link) 

- only for `arm64` devices, 28MB Download on Google Drive.
- I could also create for `arm32` devices if necessary
- please try `https://github.com/godotengine/godot-demo-projects/tree/master/misc/joypads` demo
- what is your `GUID` ?

<details><summary>Export -> Custom Template</summary>

![Image](https://github.com/user-attachments/assets/74032006-4ac4-45ad-9814-32a9c62e936f)
</details>

**Comment by dannygaray60** [comment]
Tried with my xbox series control, the view button is working, All buttons, triggers and analogs work fine! even the screenshot button is detected! The gamepad is detected as:

Xbox One Controller - 58626f7820576972656c65737320436f

I would appreciate the gesture if the fix will be integrated for godot 3.x thanks

**Comment by Alex2782** [comment]
> Xbox One Controller - 58626f7820576972656c65737320436f

Thanks for testing! 
I have this gamepad too, we still need to test other GUIDs.

overdrive on my PR [core/input/godotcontrollerdb.txt](https://github.com/godotengine/godot/pull/106021/files#diff-12390a772aca6e32f04048f16887d3e7840aad4ea9c57573f162cad917e4d4afR10-R20)
